.gitignore

```diff
@@ -5,6 +5,9 @@ __pycache__/
 # C extensions
 *.so
 
+# .gitignore repo
+joe/data
```

Comment:
Nope. This won't work.

If we do not package `joe/data/` files, the end user will have to download the files manually, which is extremely counter-intuitive.